src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -313,6 +350,7 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
 
             if (!ArmCacheEnabled)
             {
+                // TODO: Discuss if we need other sections for Arm cache is not enabled.
```

Comment:
I described what I think we need to do for this in https://github.com/Azure/azure-functions-host/pull/7527#issuecomment-967292458

Reply:
will do!

Reply:
Done